src/routes/transactions/entities/transaction-info.entity.ts

```diff
@@ -1,12 +1,16 @@
 import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { RichHumanDescriptionFragment } from '@/routes/transactions/entities/human-description.entity';
 
 export class TransactionInfo {
   @ApiProperty()
   type: string;
   @ApiPropertyOptional({ type: String, nullable: true })
-  humanDescription: string | null;
+  humanDescription: RichHumanDescriptionFragment[] | null;
```

Comment:
❓ What is the difference between `null` and an empty array for `humanDescription`?

Reply:
I don't think there is a semantic difference but if we want it to be clear I suggest we go with `null` otherwise clients would have to check the length of the array in order to be sure that it exists which is not as explicit as `null`.